modules/rhcos-restore-node-scan-with-stackrox-scanner.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/scan-rhcos-node-host.adoc
+// * cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-restore-node-scan-stackrox-scanner_{context}"]
+= Restoring {op-system} node scanning with Stackrox Scanner
+
+[role="_abstract"]
+If you use {ocp}, you can enable scanning of {op-system-first} nodes for vulnerabilities by using {rh-rhacs-first}.
+This feature is available with both: the Stackrox Scanner and Scanner V4. 
+Follow this procedure if you want to keep using Scanner V4, 
+but make the {op-system-first} node scanning feature use the Stackrox Scanner.
```

Comment:
My suggestion for wording:

Follow this procedure if you want to use the StackRox scanner to scan {op-system-first} nodes, but you want to keep using Scanner V4 to scan other nodes.

(Trying to make it a little less wordy but still get your point across - if this helps?)

Reply:
Done!